includes/class-content-toc.php

```diff
@@ -118,11 +138,12 @@ public function shortcode( $atts, $content = null ) {
 				$item_text = strip_tags( $item_match_arr[1] );
 
 				// Add filter to allow custom TOC item markup
-				$toc_list .= apply_filters(
+				$items_html .= apply_filters(
```

Comment:
Should I keep a filter here in case the link element needs to be changed? A class added or something? A bit unsure.